x-pack/plugins/ml/common/types/job_service.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Job, JobStats } from './anomaly_detection_jobs';
+
+export interface MlJobsResponse {
```

Comment:
yes, you're right.
I won't do this renaming in this PR because all of the interfaces in this file could be renamed.
I'll change them in a separate follow up.